webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue

```diff
@@ -257,8 +267,8 @@ export default {
           change: function (evt) {
             const newData = evt.editor.getData();
             self.inputVal = newData;
-            if (!self.activityId) {
-              localStorage.setItem('activity-message', newData);
+            if (!self.activityId && self.useDraftManagement && self.contextName) {
+              localStorage.setItem(`activity-message-${self.contextName}`,  JSON.stringify({'url': self.baseUrl, 'text': newData}));
```

Comment:
we should manage as well the default value to display in this component by using `localStorage.getItem...` when `this.value` is empty when calling the method `initCKEditor`

Reply:
done